### ISSUE TITLE
Excessive GPU usage in current versions.

### ISSUE THREAD

**Issue by willianholtz** [obsproject/obs-studio#10007]
### Operating System Info

Other

### Other OS

Arch Linux

### OBS Studio Version

30.0.2

### OBS Studio Log URL

https://obsproject.com/logs/avUNgS6FQJ7gtLrW

### Expected Behavior

GPU usage should be lower.

### Current Behavior

High GPU consumption

### Steps to Reproduce

I took a long, long time and avoided updating OBS on Arch Linux as much as possible, however a few days ago I was forced to do this due to several updates and this broke the version I was using, 29.0. Now with the new version, it has become almost impossible to record gameplay without experiencing excessive lag. When I used version 29.0, when opening OBS the GPU usage was between 3~4%, it rarely went up to 13%, now, only when opening it goes to 30~40%, when you start a recording the GPU is often used ranges from 45~50% usage.

I know, I should have a log of the previous version before posting here. But I don't know if it would solve it... However, only the log of the current version is here!

I looked in several places on the internet about the subject, but nothing that could help. There is a lot about this, but nothing that actually solves it.

### Anything else we should know?

Arch Linux 
64GB ram
RTX 2060 12GB
Monitor 4K
NVMe KCM3000

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request rather than a bug report, and we are not currently accepting support requests on GitHub Issues. As a note, you are using third-party plugins which might be contributing to performance issues. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for assistance.



Thank you!

**Comment by alex285** [comment]
I think it's really a bug, and actually it happens for several months;  OBS Studio maxes out the GPU on launch (and frankly it lags the full desktop);

OBS: 30.0.2 Flatpak
NVIDIA 545.29
GNOME 45 (reproducible both on Wayland and Xorg)

**Comment by Fenrirthviti** [comment]
Unfortunately, without more concrete details other than "I think this is a bug" there isn't much we can do. If you go through the support channels so someone can troubleshoot and take a closer look, this might lead to a proper bug report that we can take a closer look at. But at the moment, we don't want to do support troubleshooting for non-specific issues through GitHub.

**Comment by willianholtz** [comment]
Yes, the problem should be concrete, but there is a failure in the log. There is no way to visualize what is wrong and why the GPU is simply being used all the time at extremely high usage! I did something I didn't want to do, which was formatting my OS, and even so, no third-party plugins, no backup folder, zero, absolutely standard, and the same thing happens, starting OBS makes the GPU work above 40 %.

If there is any way we can debug this problem, let us know, because it really is complicated. And searching on Google, it's easy to find the same problem with different types of machines and OS.

We don't want to cause problems for the devs, but this has been happening for a few versions now!

@alex285 I was thinking it could be something related to my GPU (RTX 2060 and RTX 3090 on another PC) but I found several AMDs with the same problem.
